### PR TITLE
add crosschain `tokens.transfers`

### PR DESCRIPTION
**Pull request by jeff-dude**
looking to make the new token transfers spells crosschain & expose on the data explorer. we should start to consider communicating these out as new transfers spells and ask for feedback.

### CODE REVIEW

**Comment by jeff-dude** [comment]
testing CI with a smaller date range, we can revert back when/if completed

**Review by Hosuke — APPROVED** [review]

File: models/_sector/tokens/tokens_transfers.sql [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    unique_key = ['unique_key', 'blockchain'],
+    post_hook='{{ expose_spells(\'["arbitrum", "avalanche_c", "base", "bnb", "celo", "ethereum", "fantom", "gnosis", "optimism", "polygon", "zksync", "zora"]\',
+                                "sector",
+                                "tokens",
+                                \'["aalan3", "jeff-dude"]\') }}'
+)
+}}
+
+{% 
+    set blockchains = [
```
Comment (aalan3) [inline models/_sector/tokens/tokens_transfers.sql:18]: we should refer to the actual models here it makes jumping to the model easier.

File: models/_sector/tokens/tokens_transfers.sql [hunk 2]
```diff
@@ -0,0 +1,75 @@
+{{config(
+    schema = 'tokens',
+    alias = 'transfers',
+    partition_by = ['token_standard', 'block_date', 'blockchain'],
```
Comment (aalan3) [inline models/_sector/tokens/tokens_transfers.sql:4]: I think the order here matters and block_date should be last. I also think maybe we should drop `blockchain` here. If you are querying this table I'm assuming you'd want all or most chains.

File: models/_sector/tokens/tokens_transfers.sql (hunk 1 above)
Comment (jeff-dude) [reply models/_sector/tokens/tokens_transfers.sql:18]: can you clarify on "jumping to model easier"?

Comment (aalan3) [reply models/_sector/tokens/tokens_transfers.sql:18]: in vs code for example you can command click the model and go directly to the definition, it's quite nice.

File: models/_sector/tokens/tokens_transfers.sql (hunk 2 above)
Comment (jeff-dude) [reply models/_sector/tokens/tokens_transfers.sql:4]: interesting on the order, didn't realize that. i can reorder.

i think i'd push back on removing `blockchain`, as i think most queries will either filter or group by on `blockchain` on this crosschain spell. isn't that the best use case for adding a partition?

File: models/_sector/tokens/tokens_transfers.sql (hunk 1 above)
Comment (jeff-dude) [reply models/_sector/tokens/tokens_transfers.sql:18]: TIL 🤯 

i can go ahead and make that change. i like the look of this current approach, but can see the benefit of changing outweighing this

File: models/_sector/tokens/tokens_transfers.sql (hunk 2 above)
Comment (aalan3) [reply models/_sector/tokens/tokens_transfers.sql:4]: Yeah maybe you are right, we can keep it then

**Pull request merged**